docs/beast_priors.rst

```diff
@@ -136,6 +136,15 @@ The two mass function supported are:
 
   mass_prior_model = {'name': 'salpeter'}
 
+There is also a flat mass prior.  This is useful for creating grids for BEAST
+verification (see :doc:`Simulations <simulations>`), and should not be
+used for a standard fitting run.
+
+.. code-block:: python
+
+  mass_prior_model = {'name': 'flat_linear'}
+
+
```

Comment:
Great.  The docs are here already.  Yeah.  I think you could just call this option `flat` as all the IMFs are evaluated in linear mass.  Saying `flat_linear` implies that linear is not always the case.

Reply:
Since the age priors distinguish between `flat_linear` and `flat_log`, I thought it would be good to be explicit.  But perhaps that's more of a thing for ages than masses?